src/components/BackOffice/ContractsDashboard/Contractor/ContractorView.tsx

```diff
@@ -57,7 +57,7 @@ const ContractorView = ({
   )
 
   const descendingDateContractsWithSorCode = useMemo(
-    () => (contractsWithSorCode ? [...contractsWithSorCode].reverse() : null),
+    () => contractsWithSorCode && [...contractsWithSorCode].reverse(),
```

Comment:
![image](https://github.com/user-attachments/assets/40d84014-d2fb-45ae-8508-5280bc4a159d)

This will still return `null` when `contractsWithSorCode` is null. If null is unwanted, you'll probably want something like:

`contractsWithSorCode ? [...contractsWithSorCode].reverse() : []`